tests/foreman/api/test_docker.py

```diff
@@ -1278,77 +1288,82 @@ def test_positive_create_using_cv(self):
         self.assertEqual(len(content_view.version), 1)
         cvv = content_view.read().version[0].read()
         promote(cvv, lce.id)
-        for compute_resource in (self.cr_internal, self.cr_external):
-            with self.subTest(compute_resource):
+
+        # publishing takes few seconds sometimes
+        retries = 10 if bz_bug_is_open(1452149) else 1
+        for i in range(retries):
```

Comment:
not sure about whole block, maybe we can actually check whether necessary resource is available and then do create once. Also, not an expert in python, but `if i == retries - 1:` looks a little bit weird